### PR TITLE
Improve test coverage through integration tests

### PR DESCRIPTION
**Pull request by trieloff**
This has been a bit more painful than I expected. 
The integration test now loads helix-rum-enhancer through
a (slightly prepared) version of helix-rum-js and then
validates the checkpoints that have been sent using a mocked
`navigator.sendBeacon`. This helps to bring up the coverage
of `index.js`, but there are still good chunks of it that
require more fiddling, especially when we look at URL parameter
comprehension

- **test(index): click somewhere**
- **test(integration): perform integration tests with slightly patched rum-js**
- **test(webkit): make integration test pass in webkit**
- **test(integration): make sure correct code coverage for integration test gets generated**
- **test(integration): remove unused plugins**
- **test(integration): fake more sendBeacons**
- **test(integration): test more checkpoints**

### CODE REVIEW

File: test/it/fromrum.test.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    };
+  </script>
+  <script defer type="text/javascript" src="/.rum/@adobe/helix-rum-js@^2/dist/rum-standalone.js"></script>
+</head>
+
+<body>
+  <img src="/test/it/img.test.html">
+  <script type="module">
+    import { runTests } from '@web/test-runner-mocha';
+    import { sendMouse } from '@web/test-runner-commands';
```
Comment (ekremney) [inline test/it/fromrum.test.html:33]: @kptdobe referring to the discussion we had back in rumathon, maybe this `sendMouse` command what we needed for the INP test?

Comment (kptdobe) [inline test/it/fromrum.test.html:33]:
```suggestion
```

Comment (kptdobe) [reply test/it/fromrum.test.html:33]: Not needed, no ?

Comment (kptdobe) [reply test/it/fromrum.test.html:33]: To be tested, yes.

**Review by kptdobe — DISMISSED** [review]
Nice!

File: test/it/fromrum.test.html (hunk 1 above)
Comment (trieloff) [reply test/it/fromrum.test.html:33]: You got me. I'm clicking now.

Comment (trieloff) [reply test/it/fromrum.test.html:33]: The challenge may be that we have to navigate away from the page and that can interfere with recording the test results.

**Pull request merged**